### PR TITLE
i18n(ES) change Lenguage to Idioma

### PR DESCRIPTION
**Pull request by joebordes**
Idioma seems better suited but this called my attention because Lenguage is misspelled, it should be Lenguaje

### CODE REVIEW

**Comment by thc202** [comment]
Translations need to be done through Crowdin: https://crowdin.com/project/owasp-zap
The file will be overwritten in the next sync.

**Comment by joebordes** [comment]
done

**Comment by thc202** [comment]
Thank you!

File: zap/src/main/dist/lang/Messages_es_ES.properties [hunk 1]
```diff
@@ -2199,7 +2199,7 @@ target.allInScope=Todo lo que se encuentre en el alcance
 target.empty=Vac\u00edo
 
 technologies.db=DB
-technologies.lang=Lenguage
+technologies.lang=Idioma
```
Comment (thc202) [inline zap/src/main/dist/lang/Messages_es_ES.properties:2202]: Just a note, I don't think Idioma fits here, this is for programming languages.

**Pull request closed without merging**